src/ProjectionMapping/ProjectionDeckMap/layers.js

```diff
@@ -38,8 +38,20 @@ export const createGeoJsonLayer = (i, layer, GEOGRID) =>
     extruded: true,
     lineWidthScale: 20,
     lineWidthMinPixels: 2,
-    getLineColor: [255, 255, 255],
-    getFillColor: [200, 200, 200],
+    getLineColor: f => {
```

Comment:
could you define a separate function for the hex to rgba mapping? It’s used twice here and could be useful in other layers